Mathlib/MeasureTheory/Constructions/BorelSpace/Metrizable.lean

```diff
@@ -5,14 +5,14 @@ Authors: Floris van Doorn
 -/
 import Mathlib.MeasureTheory.Constructions.BorelSpace.Basic
 import Mathlib.Topology.Metrizable.Basic
+import Mathlib.Topology.IndicatorConstPointwise
```

Comment:
This added import is light. The new file is less than 75 lines and should not import anything new transitively.